README.md

```diff
@@ -158,6 +155,9 @@ There is a file `kurl-installer.yaml` that can be used to manage [kurl.sh](https
 replicated installer create --auto
```

Comment:
From above: `1. Remove the templates and values from this chart, and just use this repo as an umbrella chart that includes your main chart as a helm `dependency` in `Chart.yaml` (run `helm dependency update` before `helm package . -d manifests/`)`

I thought this new version wouldn't require removing templates and values from the chart. does this need updating too?